oppgave/src/main/java/no/nav/k9/oppgave/Oppgave.java

```diff
@@ -0,0 +1,216 @@
+package no.nav.k9.oppgave;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import no.nav.k9.oppgave.bekreftelse.Bekreftelse;
+import no.nav.k9.søknad.Innsending;
+import no.nav.k9.søknad.JsonUtils;
+import no.nav.k9.søknad.felles.DtoKonstanter;
+import no.nav.k9.søknad.felles.Kildesystem;
+import no.nav.k9.søknad.felles.Versjon;
+import no.nav.k9.søknad.felles.personopplysninger.Søker;
+import no.nav.k9.søknad.felles.type.Språk;
+import no.nav.k9.søknad.felles.type.SøknadId;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+import java.util.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.NONE, getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE)
+public class Oppgave implements Innsending {
+
+    @Valid
+    @NotNull
+    @JsonProperty(value = "søknadId", required = true)
```

Comment:
Burde kanskje vært oppgaveId her?

Reply:
Innsending har søknadId, så eg beholdt det som søknadId for å knytt det meir til interfacet